### PR TITLE
fix(toggle default value):

### PR DESCRIPTION
**Pull request by daniel7764**
-Toggle is now empty by default and not false

### CODE REVIEW

File: client/src/commons/Toggle/Toggle.tsx [hunk 1]
```diff
@@ -18,13 +18,13 @@ const Toggle: React.FC<Props> = (props: Props): JSX.Element => {
         <ToggleButtonGroup value={value} exclusive {...rest}>
             <ToggleButton className={classes.toggle} 
                 disabled={disabled}
-                style={!value ? activeButtonStyle : {}}
+                style={(!value && value !== null) ? activeButtonStyle : {}}
```
Comment (MayaSasson) [inline client/src/commons/Toggle/Toggle.tsx:21]: You should check value === false

File: client/src/commons/Toggle/Toggle.tsx [hunk 2]
```diff
@@ -18,13 +18,13 @@ const Toggle: React.FC<Props> = (props: Props): JSX.Element => {
...
                 value={false}>
                 {firstOption ? firstOption : 'לא'}
             </ToggleButton>
             <ToggleButton className={classes.toggle}
                 disabled={disabled}
-                style={value ? activeButtonStyle : {}}
+                style={(value && value !== null) ? activeButtonStyle : {}}
```
Comment (MayaSasson) [inline client/src/commons/Toggle/Toggle.tsx:27]: You should check value === true

**Review by ItayBenMoshe — APPROVED** [review]

File: client/src/commons/Toggle/Toggle.tsx (hunk 1 above)
Comment (daniel7764) [reply client/src/commons/Toggle/Toggle.tsx]: fixed

File: client/src/commons/Toggle/Toggle.tsx (hunk 2 above)
Comment (daniel7764) [reply client/src/commons/Toggle/Toggle.tsx]: fixed

**Review by NaamanTsur — APPROVED** [review]

**Review by doronlnd84 — APPROVED** [review]

**Pull request merged**